src/visualizations/formulaic_visuals.py

```
@@ -1,8 +1,10 @@
+import math
+from typing import Any, Dict
```

Ruff (0.14.3)

`typing.Dict` is deprecated, use `dict` instead

(UP035)